Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/ViewUpdaters.fs

```diff
@@ -723,4 +730,20 @@ module ViewUpdaters =
         | ValueSome prevValue, ValueSome currValue when prevValue = currValue -> ()
         | ValueNone, ValueNone -> ()
         | _, ValueSome currValue -> Element.SetMenu(target, currValue.Create() :?> Menu)
-        | ValueSome _, ValueNone -> Element.SetMenu(target, null)
\ No newline at end of file
+        | ValueSome _, ValueNone -> Element.SetMenu(target, null)
+
+    let updateIndicatorViewRef (prevValueOpt: ViewRef<CustomCarouselView> voption) (currValueOpt: ViewRef<CustomCarouselView> voption) (target: Xamarin.Forms.IndicatorView) =
+        match prevValueOpt, currValueOpt with
+        | ValueSome prevValue, ValueSome currValue when prevValue = currValue -> ()
+        | ValueNone, ValueNone -> ()
+        | _, ValueSome currValue -> 
+            match currValue.TryValue with
+            | Some v -> IndicatorView.SetItemsSourceBy(target, v)
+            | None -> ()
```

Comment:
Maybe we should set the ItemsSource to `null` in the case the ref is invalid, instead of doing nothing and keeping a previous state no longer wanted?